gce/survival-training/gce/tf-estimator-startup.sh

```diff
@@ -29,9 +29,9 @@ KEEP_ALIVE=$(curl http://metadata.google.internal/computeMetadata/v1/instance/at
 
 cd "/home/$GCE_USER"
 
-gcloud source repos clone "$TRAINER_REPO"
+git clone $TRAINER_GIT_PATH trainer
```

Comment:
@amygdala , @dizcology , @puneith : Considered your feedback very carefully and think you guys are right on all counts - source repos is probably the most confusing thing that I could have plugged in as the delivery mechanism.

In the tf-estimator example, I have replaced it with just GitHub. ( @dizcology : To respond to your question about tagging - Here, for example, it would look like checkout out a specific git tag or branch. )

For the sklearn example (not in scope of this PR), let us use GCS.